CityOfDetroit/bloom
# Remove the eligibility section of the listing view.

# Pull Request Template

## Issue

Closes #349.

## Description

Remove eligibility section of the detailed listing view.

## How Can This Be Tested/Reviewed?

View various test listings in both views and notice that the eligibility section is no longer there.


Before:
![eligibility-before](https://user-images.githubusercontent.com/4317058/129956632-8c8edbd6-a65d-42b0-b96b-07f49461736e.png)

After:
![eligibility-after](https://user-images.githubusercontent.com/4317058/129956639-ebca628b-4f0a-43e0-8e2e-cc28f7e1d51a.png)

## Checklist:
